.yamllint

```diff
@@ -32,7 +32,7 @@ rules:
   new-lines: "enable"
   octal-values: "disable"
   quoted-strings:
-    quote-type: "double"
+    quote-type: "any"
```

Comment:
I am definitely against this adjustment. Our .yaml files follow the Home Assistant Style Guide for .yaml files, which prescribes double-quoted strings. I see no reason why we should not apply this to the rest of the .yaml files in the repo as well

Reply:
Check, will reverse it.